Portfolio/index.html

```diff
@@ -4,27 +4,33 @@
     <title>Portfolio</title>

```

Comment:
Consider adding integrity and crossorigin attributes for enhanced security when linking to external stylesheets.